### PR TITLE
Removes BS key from basic Marshal/Ranger headsets

### PR DESCRIPTION
**Pull request by SingingSpock**
## About The Pull Request
<details>
<summary>
	Removes BS key from Marshal Officer/Ranger headsets
</summary>
<hr>

This is intended to reduce the constant flow of Marshals to the gate whenever there's a whiff of trouble even when Blackshield has enough personnel to handle it. If Blackshield needs help they can call for it on common. Specialist still keeps both since they're supposed to help supply BS as well as Marshals, and WO keeps both because they're a Head and (supposedly) not as likely to just jump into other people's jurisdictions.	
<hr>
</details>

## Changelog
:cl:
tweak: Blackshield key removed from Marshal Officer and Ranger Headsets
add: Supply Specialist headsets that have Blackshield and Marshal keys
/:cl:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -226,3 +226,5 @@ aux*.pdb
 /artifacts
 
 tgui/public/
+
+maps/__Nadezhda/map/Holding.dmm
```
Comment (Trilbyspaceclone) [inline .gitignore:230]: ?

**Comment by Trilbyspaceclone** [comment]
Im actually insanely against this pr
marshals should have blackshield coms all around

**Comment by SingingSpock** [comment]
> Im actually insanely against this pr marshals should have blackshield coms all around

For what purpose? Blackshield troopers and corpsmen do not have Marshal comms, why should Marshals and Rangers have Blackshield comms?

**Comment by Trilbyspaceclone** [comment]
Marshals can ball blackshield over with it
and blackshield can talk to marshals with it about sus things and law

its insainly usefull as a marshal to have it, and for blackshield to be able to talk to marshals

**Comment by cdb-is-not-good** [comment]
The issues tend not to arise from officers arriving at shield jurisdiction but WO/ranger/specs who keep the radio access in this PR. If you want the marshals to step off, reaffirm shield jurisdiction at the gate in a more clear way. We've already had one incident where a WO tried to claim jurisdiction at the gate "because it's the inside edge"

**Comment by benj8560** [comment]
Not at all a fan. These two departments being able to communicate privately is extremely important.  Blackshield needs to be more assertive if they let Marshals walk all over them that's their own fault.

**Comment by cdb-is-not-good** [comment]
To be fair the blackshield enforcing their jurisdiction is a tough one because it's not THAT well defined beyond "the perimeter". We've dead ass had WOs insist that the inside of the perimeter regardless how close is their jurisdiction(it is not, and never has been), but it needs to probably be clarified.

**Comment by Trilbyspaceclone** [comment]
Seems more of an ic issue with folks being passive rather then a code fixable problem. If marshals abuse it more-so / actively then this can be revisited

**Pull request closed without merging**